docker-compose.test.yaml

```diff
@@ -10,6 +10,8 @@ services:
       target: development
     image: ffc-demo-web-development
     command: npm test
+    environment:
+      NODE_ENV: test
```

Comment:
Previously work was done to remove the need for running Redis during tests.
This got lost somewhere along the way, so as I was updating the test compose files I removed the unwanted Redis container from the top level and pushed it down to only where it is needed, and removed duplication in the comose file.